.github/workflows/scripts/istio_deploy.sh

```diff
@@ -1,21 +1,23 @@
 #!/usr/bin/env bash
-
 # This script is used to deploy Istio on Kubernetes
 #
 # Also deploys the bookinfo application on Istio and passes the gateway URL to Meshery
-
 # See: https://github.com/service-mesh-performance/service-mesh-performance/blob/master/protos/service_mesh.proto
+
 export MESH_NAME='Istio'
 export SERVICE_MESH='ISTIO'
 
-curl -L https://istio.io/downloadIstio | ISTIO_VERSION=1.11.4 sh -
-cd istio-1.11.4
-export PATH=$PWD/bin:$PATH
-istioctl install --set profile=demo -y
-kubectl label namespace default istio-injection=enabled
+# Check if mesheryctl is present, else install it
+if ! [ -x "$(command -v mesheryctl)" ]; then
+    echo 'mesheryctl is not installed. Installing mesheryctl client... Standby... (Starting Meshery as well...)' >&2
+    curl -L https://meshery.io/install | ADAPTERS=istio PLATFORM=kubernetes bash -
+fi
 
-kubectl apply -f samples/bookinfo/platform/kube/bookinfo.yaml
-kubectl apply -f samples/bookinfo/networking/bookinfo-gateway.yaml
+sleep 10
+#mesheryctl system login --provider None
+echo 'E' | mesheryctl mesh deploy adapter meshery-istio:10000 --token "./.github/workflows/auth.json"
```

Comment:
Probably the Istio didn't deploy completely, can we add a `sleep` and a check here?

Reply:
Istio is deployed, I found a possible cause now, size of minikube cluster isn't sufficient, I'll change size and check